.github/workflows/ci.yml

```diff
@@ -106,3 +106,39 @@ jobs:
           . ./instrument.sh
           dotnet help
             test "$(ls -A '${{ matrix.log-dir }}' )"
+
+  pack:
+    needs: build
+
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3.1.0
+      - uses: nuget/setup-nuget@v1.0.7
+
+      - name: Download Windows Artifacts from build job
+        uses: actions/download-artifact@v3
+        with:
+          name: bin-windows-2022
+          path: nuget/bin-windows-2022
+
+      - name: Download Ubuntu Artifacts from build job
+        uses: actions/download-artifact@v3
+        with:
+          name: bin-ubuntu-20.04
+          path: nuget/bin-ubuntu-20.04 
```

Comment:
I suggest changing the path to `nuget/bin-linux-glibc` for forward compatibility.